prolude.cabal

```
@@ -34,8 +35,9 @@ library
 
     build-depends: 
         base >= 4.13.0 && < 4.15
-        , aeson >= 1.4.7 && < 1.5
+        , aeson >= 1.4.7 && < 1.6
         , bytestring >= 0.10.10 && < 0.11
+        , containers >= 0.6.2 && < 0.6.7
```

```suggestion
        , containers >= 0.6.2 && < 0.7
```

As a general rule, your constraints should look like `>= x.y.z && < x.y`. https://pvp.haskell.org/#dependencies-in-cabal